app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+
+    if len(command.split()) == 3 and command.split()[0] == "cp":
+        cd, file_name, new_file = command.split()
```

Comment:
```suggestion
    parts = command.split()
    if len(parts) == 3 and parts[0] == "cp":
        cd, file_name, new_file = parts
```
Better to do splitting one time instead of 3